pipeline_steps/pubcloud.groovy

```diff
@@ -92,27 +82,22 @@ String getPubCloudSlave(Map args){
       if (!("inventory" in args)){
         args.inventory = "inventory"
       }
-      withCredentials(common.get_cloud_creds()){
```

Comment:
Is it the removal of this that's causing https://rpc.jenkins.cit.rackspace.net/job/RE-unit-test-phobos-vpn/992/ to fail?

Reply:
wrapped with `withRequestedCredentials("cloud_creds") {}` block.

Reply:
recheck_unit_tests